packages/x-tree-view-pro/src/internals/plugins/useTreeViewItemsReordering/useTreeViewItemsReordering.ts

```diff
@@ -136,47 +131,36 @@ export const useTreeViewItemsReordering: TreeViewPlugin<UseTreeViewItemsReorderi
 
   const startDraggingItem = React.useCallback(
     (itemId: string) => {
-      store.update((prevState) => {
-        const isItemBeingEditing = selectorIsItemBeingEdited(prevState, itemId);
-        if (isItemBeingEditing) {
-          return prevState;
-        }
+      const isItemBeingEditing = labelSelectors.isItemBeingEdited(store.state, itemId);
+      if (isItemBeingEditing) {
```

Comment:
Unrelated but maybe:

```suggestion
      const isItemBeingEdited = labelSelectors.isItemBeingEdited(store.state, itemId);
      if (isItemBeingEdited) {
```